connectors/src/connectors/zendesk/index.ts

```diff
@@ -86,15 +105,55 @@ export class ZendeskConnectorManager extends BaseConnectorManager<null> {
   }
 
   async stop(): Promise<Result<undefined, Error>> {
-    throw new Error("Method not implemented.");
+    return stopZendeskSyncWorkflow(this.connectorId);
   }
 
   async resume(): Promise<Result<undefined, Error>> {
-    throw new Error("Method not implemented.");
+    const { connectorId } = this;
+    const connector = await ConnectorResource.fetchById(connectorId);
+    if (!connector) {
+      logger.error({ connectorId }, "[Zendesk] Connector not found.");
+      return new Err(new Error("Connector not found"));
+    }
+
+    const dataSourceConfig = dataSourceConfigFromConnector(connector);
+    try {
```

Comment:
Why do we need the try/catch if we have a `Result` type? Also in many other places there is no `try/catch` that wraps this call.

Reply:
my bad, changed that in `launchZendeskSyncWorkflow` a while ago and got a little bit confused here

Reply:
goal here is indeed to always use the `Result` type by catching it if necessary in the deepest level possible and return it in `launchZendeskSyncWorkflow` and then in `resume`.

Reply:
done